pkg/adapter/incoming.go

```diff
@@ -60,6 +60,7 @@ func (l *listener) detectIncoming(ctx context.Context, req *http.Request, payloa
 		Name:      hook.Secret.Name,
 		Key:       hook.Secret.Key,
 	}
+	l.logger = l.logger.With("namespace", repo.Namespace)
```

Comment:
here too l.logger is shared
better to create to assign to a new logger instance and use it

Reply:
its shared but later value is overridden here https://github.com/openshift-pipelines/pipelines-as-code/pull/1251/files#diff-c985856260500332bdba26c6f6c75a2e3c365e0ecd4272b92762e5de820dc2ccR62 

So did not create instance